client/landing/stepper/declarative-flow/internals/steps-repository/goals/goals.tsx

```diff
@@ -2,35 +2,35 @@ import { Onboard } from '@automattic/data-stores';
 import { useTranslate } from 'i18n-calypso';
 import type { Goal } from './types';
 
-const GoalKey = Onboard.GoalKey;
+const SiteGoal = Onboard.SiteGoal;
```

Comment:
For future work, I would recommend keeping refactoring work (e.g. variable renames, cleanups, etc.) on a separate PR to make things easy to review.